tests/xrt/100_ert_ncu/xrtxx-ip.cpp

```diff
@@ -115,13 +117,15 @@ struct job_type
       auto grpid1 = kernel.group_id(1);
 
       const size_t data_size = ELEMENTS * ARRAY_SIZE;
-      a = xrt::bo(device, data_size*sizeof(unsigned long), grpid0);
+      //error from below: call of overloaded ‘bo(const xrt::device&, long unsigned int, int&)’ is ambiguous
+      //a = xrt::bo(device, data_size*sizeof(unsigned long), grpid0);
+      a = xrt::bo(device, data_size*sizeof(unsigned long), xrt::bo::flags::normal, grpid0);
```

Comment:
Please show me where and how to get this "ambiguous" error.

Reply:
ignore. not seen now